app/controllers/application_controller.rb

```diff
@@ -6,7 +6,12 @@ class ApplicationController < ActionController::Base
 
   # Run authorization on all actions
   # check_authorization
-  protect_from_forgery
+  # np: Rails 4 API would let us do this: http://bit.ly/1HokQ0f 
+  protect_from_forgery with: :exception
+  # ⬆ doesn't seem to have any effect in rails 3.  We still return 200.
+  # rails 3 assumption is that the controller would check the user
+  # We could do eg in controllers/embeddable/open_responses_controller.rb : 
+  # http://bit.ly/1I8FxAw (github)
```

Comment:
It is possible to do this globally by adding a method to the application controller:
https://github.com/concord-consortium/lara/blob/fa72fc3f027243a8c390f9bda49c84391950a552/app/controllers/application_controller.rb#L311-L314
That method could throw an exception instead of logging and reseting the session.